scripts/aws_bastion_mfa_export.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+while getopts p: flag
+do
+    case "${flag}" in
+        p) profile=${OPTARG};;
+    esac
+done
+
+echo "|_ Using profile flag: $profile";
+
+# Make sure we have a temporary token
+# This will also request your MFA token if needed
+echo "|_ Requesting identity with profile $profile"
+aws sts get-caller-identity --profile $profile
```

Comment:
I'm a big fan of shellcheck so

^------^ SC2086: Double quote to prevent globbing and word splitting

Change $profile to $"profile"